en/docs/install-and-setup/manual.md

```diff
@@ -10,14 +10,37 @@
 ### Installation Steps
 1. Extract WSO2 APIM product. Let's call it `<WSO2_APIM_HOME>`.
 2. Navigate to `<WSO2_APIM_HOME>/bin` directory and execute the update command based on the operating system to bring WSO2 API Manager up to date by running the [Update Tool](https://updates.docs.wso2.com/en/latest/updates/update-tool/). 
-3. Extract WSO2 OH APIM Accelerator to `<WSO2_APIM_HOME>`. Let's call it `<WSO2_OH_APIM_ACC_HOME>`.
-4. [Optional] Check the accelerator configurations in <WSO2_HC_APIM_ACC_HOME>/conf/config.toml file to enable or disable features.
-5. Navigate to `<WSO2_OH_APIM_ACC_HOME>` directory and execute following command. This will copy the artifacts to the WSO2 APIM and add the required configurations.
+3. You can try the following commands based on your Operating system to update the WSO2 API Manager. As an example, if you are Mac user, we need to execute the following commands.
+
+    Provide your wso2 credentials and update the tool.  
+    ```
+    ./wso2update_darwin
+    ```
+
+    Check the latest update level for the product. 
+    ```
+    ./wso2update_darwin check
+    ```
+
+    Update the product to the update level you need. As an example, if you need to update it to wso2am-4.2.0.118 we can provide the command as below. 
```

Comment:
Do we need to mention a product update level? Mentioning update using update tool to the latest is enough right?

Reply:
Fixed in https://github.com/wso2/open-healthcare-docs/pull/44/commits/265eb2611a3f4cda6c4e230c05fb3f442d5afcee